app/scripts/templates/settings/clients.mustache

```diff
@@ -26,8 +26,7 @@
             {{^isCurrentDevice}}
             <div class="last-connected">{{lastAccessTimeFormatted}}</div>
             {{/isCurrentDevice}}
-            {{#isDevice}}<button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#t}}Disconnect…{{/t}}</button>{{/isDevice}}
-            {{^isDevice}}<button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#t}}Disconnect{{/t}}</button>{{/isDevice}}
+            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#isDevice}}{{#t}}Disconnect…{{/t}}{{/isDevice}}{{^isDevice}}{{#t}}Disconnect{{/t}}{{/isDevice}}</button>
```

Comment:
@divyabiyani this line seems way too long now, we should probably line break before `{{#isDevice}}` and clean it up.